public/data/AsifHabib.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Asif Habib",
+    "bio": "Web Developer | Devops Enthusiast",
+    "avatar": "https://github.com/asif032000.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/asif032000",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AsifHab12532650",
+        "icon": "twitter"
+      }, 
+      {
+          "name": "Follow me on LinkedIn", 
+          "url": "https://www.linkedin.com/in/asif-habib-aaaa021a2/", 
+          "icon": "linkedin"
+      }
+    ]
+  }
```

Comment:
GitHub is highlighting that this needs an extra blank line - we can merge and this can be fixed later